### PR TITLE
fix broken links + add workflow to catch future broken links

### PR DESCRIPTION
**Pull request by lbarraga**
This PR contains a scripts that filters a-tags out of the HTML in the build files and checks if they exist. For more info check the included README

### CODE REVIEW

**Comment by boegel** [comment]
@lbarraga Can you also add the whitelist file that makes sense (for example the `account.vscentrum.be` URLs, for which we get HTTP status code 500 since we're not logged in), and a GitHub Actions workflow to run this link checker script (I'm guessing it will have to be included as an extra step in the existing build workflow, since we need to built HTML)

**Comment by lbarraga** [comment]
@boegel Ready for review

File: scripts/link_checker/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+The link checker consists of two parts:
+- A bash script that outputs all links from the HTML files in the build directory along with the path to the file containing the link.
+- A Python script that reads the output of the bash script and checks the status code of each link.
+
+The Python script will output all links that return a status code other than 200 (OK) along with the path to the file containing the link.
+
+## Usage
+
+```shell
+$ ./link_checker.sh /path/to/mkdocs/build > links.txt
```
Comment (boegel) [inline scripts/link_checker/README.md:12]:
```suggestion
$ ./generate_links_from_build.sh /path/to/mkdocs/build > links.txt

File: scripts/link_checker/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+```shell
+$ python link_checker.py [--whitelist <whitelist.txt>] links.txt
```
Comment (boegel) [inline scripts/link_checker/README.md:16]:
```suggestion
$ python check_links.py [--whitelist <whitelist.txt>] links.txt

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@lbarraga Your link checker works really well, https://wiki.openfoam.com/Tutorials seems to be broken now :)

**Comment by lbarraga** [comment]
only had to re-run the build. Link seems ok now.

**Pull request merged**